examples/cg_possion.py

```diff
@@ -6,10 +6,10 @@
 steps = 32
 eps = 1e-5
 
-b = ti.field(ti.f32, (n, n))
-x = ti.field(ti.f32, (n, n))
-d = ti.field(ti.f32, (n, n))
-r = ti.field(ti.f32, (n, n))
+b = ti.field(float, (n, n))
```

Comment:
simply have a tiny question ;-).

Has the type system `ti.xxx` been not recommened anymore in Taichi-scope? I checked the [doc](https://taichi.readthedocs.io/en/stable/type.html), it seems here is not evidence.

Reply:
Sorry about the doc, please checkout the [Default Precisions](https://taichi.readthedocs.io/en/stable/type.html#default-precisions) chapter:
```py
ti.init(default_ip=ti.i64, default_fp=ti.f32)  # set default precisions on ti.init()

x = ti.var(float, 5)
y = ti.var(int, 5)
# is equivalent to:
x = ti.var(ti.f32, 5)
y = ti.var(ti.i64, 5)

def func(a: float) -> int:  # use float and int as aliases for default precision
    ...

# is equivalent to:
def func(a: ti.f32) -> ti.i64:
    ...
```
For example, users can run `TI_DEFAULT_FP=64 python examples/fractal.py` for running fractal in float64 precision without changing it's source code.

Reply:
Thanks! I get it. It's a lot of convenience, that we don't need to specify `ti.xxx` everytime. We can recommend our users  do coding in this way as well.

Reply:
Yes, feel free to advocate it more in doc if you'd like to.